plugin.video.youtube/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.youtube" name="YouTube" version="5.1.17" provider-name="bromix">
+<addon id="plugin.video.youtube" name="YouTube" version="5.2.1" provider-name="h0d3nt3uf3l, jdf76, bromix">
```

Comment:
Well, it's up to you what you do. But I would choose a construct, that is simple and thus easily understandable. You don't want to confuse potential committers or people interested in the addon. Or render them unable to find your repo ;)
If you go for it, it's probably nothing that has to be done in this release.